### PR TITLE
feat: import parquet to TiDB

### PR DESCRIPTION
**Pull request by buchuitoudegou**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@frank.shen@pingcap.com

File: migrate-from-parquet-files-to-tidb.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: 从 Parquet 文件迁移数据到 TiDB
+summary: 介绍如何使用 TiDB Lightning 从 Parquet 文件迁移数据到 TiDB。
+---
+
+# 从 SQL 文件迁移数据到 TiDB
+
+本文介绍如何使用 TiDB Lightning 从 Apache Parquet 文件迁移数据到 TiDB，以及如何生成 Parquet 文件。
```
Comment (Frank945946) [inline migrate-from-parquet-files-to-tidb.md:8]: 这边加个限制，是“ Apache Hive 导出的Parquet 文件”

File: migrate-from-parquet-files-to-tidb.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+{{< copyable "" >}}
+
+```toml
+[lightning]
+# 日志
+level = "info"
+file = "tidb-lightning.log"
+
+[tikv-importer]
+# "local"：默认使用该模式，适用于 TiB 级以上大数据量，但导入期间下游 TiDB 无法对外提供服务。
```
Comment (Frank945946) [inline migrate-from-parquet-files-to-tidb.md:64]: “"local"：默认使用该模式，适用于 TiB 级以上大数据量，但导入期间下游 TiDB 无法对外提供服务。” 从6.2版本之后不是把影响范围降低到了表级别了么？

File: migrate-from-parquet-files-to-tidb.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+# 源数据目录
+data-source-dir = "${data-path}" # 本地或 S3 路径，例如：'s3://my-bucket/sql-backup'
+
+[tidb]
+# 目标集群的信息
+host = ${host}                # 例如：172.16.32.1
+port = ${port}                # 例如：4000
+user = "${user_name}"         # 例如："root"
+password = "${password}"      # 例如："rootroot"
+status-port = ${status-port}  # 导入过程 Lightning 需要在从 TiDB 的“状态端口”获取表结构信息，例如：10080
```
Comment (Frank945946) [inline migrate-from-parquet-files-to-tidb.md:80]: 10080是默认端口么？建议改成默认端口

File: migrate-from-parquet-files-to-tidb.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+pd-addr = "${ip}:${port}"     # 集群 PD 的地址，Lightning 通过 PD 获取部分信息，例如 172.16.31.3:2379。当 backend = "local" 时 status-port 和 pd-addr 必须正确填写，否则导入将出现异常。
+```
+
+关于配置文件更多信息，可参阅 [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md)。
+
+## 第 5 步：执行导入
+
+运行 `tidb-lightning`。如果直接在命令行中启动程序，可能会因为 `SIGHUP` 信号而退出，建议配合 `nohup` 或 `screen` 等工具。
+
+若从 Amazon S3 导入，则需将有权限访问该 S3 后端存储的账号的 SecretKey 和 AccessKey 作为环境变量传入 Lightning 节点。
```
Comment (Frank945946) [inline migrate-from-parquet-files-to-tidb.md:90]: 不能通过填写url和role-arn来访问么？类似云上的模式

**Review by Frank945946 — APPROVED** [review]

File: migration-tools.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ TiDB 提供了丰富的数据迁移相关的工具，用于全量迁移、增量
 |:---|:---|:---|:---|:---|:---|
 |  [TiDB DM](/dm/dm-overview.md)|用于将数据从与 MySQL 协议兼容的数据库迁移到 TiDB。  |  MySQL，MariaDB，Aurora，MySQL| TiDB   | 一体化的数据迁移任务管理工具，支持全量迁移和增量同步；支持对表与操作进行过滤；支持分库分表的合并迁移。 |  建议用于 1TB 以内的存量数据迁移。|
 | [Dumpling](/dumpling-overview.md) | 用于将数据从 MySQL/TiDB 进行全量导出。| MySQL，TiDB| SQL，CSV  | 支持全新的 table-filter，筛选数据更加方便；支持导出到 Amazon S3 云盘|如果导出后计划往非 TiDB 的数据库恢复，建议使用 Dumpling；如果是往另一个 TiDB 恢复，建议使用 BR。 |
-| [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)| 用于将数据全量导入到 TiDB。|Dumpling 输出的文件；CSV 文件；从本地盘或 Amazon S3 云盘读取数据。 | TiDB | 支持迅速导入大量新数据，实现快速初始化 TiDB 集群的指定表；支持断点续传；支持数据过滤。| 如果使用 Local-backend 进行数据导入，TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。如果你不希望 TiDB 集群的对外服务受到影响，可以参考 TiDB Lightning TiDB-backend 中的硬件需求与部署方式进行数据导入。|
+| [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)| 用于将数据全量导入到 TiDB。|Dumpling 输出的文件；CSV 文件；从 Hive 或者 Aurora 导出的 parquet；从本地盘或 Amazon S3 云盘读取数据。 | TiDB | 支持迅速导入大量新数据，实现快速初始化 TiDB 集群的指定表；支持断点续传；支持数据过滤。| 如果使用 Local-backend 进行数据导入，TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。如果你不希望 TiDB 集群的对外服务受到影响，可以参考 TiDB Lightning TiDB-backend 中的硬件需求与部署方式进行数据导入。|
```
Comment (Oreoxmt) [inline migration-tools.md:18]:
```suggestion
| [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)| 用于将数据全量导入到 TiDB。|Dumpling 输出的文件；CSV 文件；从 Hive 或者 Aurora 导出的 Parquet 文件；从本地盘或 Amazon S3 云盘读取数据。 | TiDB | 支持迅速导入大量新数据，实现快速初始化 TiDB 集群的指定表；支持断点续传；支持数据过滤。| 如果使用 Local-backend 进行数据导入，TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。如果你不希望 TiDB 集群的对外服务受到影响，可以参考 TiDB Lightning TiDB-backend 中的硬件需求与部署方式进行数据导入。|
```

File: migrate-from-parquet-files-to-tidb.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
+---
+title: 从 Parquet 文件迁移数据到 TiDB
+summary: 介绍如何使用 TiDB Lightning 从 Parquet 文件迁移数据到 TiDB。
+---
+
+# 从 SQL 文件迁移数据到 TiDB
```
Comment (buchuitoudegou) [inline migrate-from-parquet-files-to-tidb.md:6]:
```suggestion
# 从 Parquet 文件迁移数据到 TiDB
```

File: migrate-from-parquet-files-to-tidb.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
+---
+title: 从 Parquet 文件迁移数据到 TiDB
+summary: 介绍如何使用 TiDB Lightning 从 Parquet 文件迁移数据到 TiDB。
+---
+
+# 从 Parquet 文件迁移数据到 TiDB
+
+本文介绍如何使用 TiDB Lightning 从 Apache Parquet 文件迁移数据到 TiDB，以及如何生成 Parquet 文件。
```
Comment (buchuitoudegou) [inline migrate-from-parquet-files-to-tidb.md:8]:
```suggestion
本文介绍如何使用 TiDB Lightning 从 Apache Hive 导出的 Parquet 文件迁移数据到 TiDB，以及如何生成 Parquet 文件。
```

File: migrate-from-parquet-files-to-tidb.md (hunk 4 above)
Comment (buchuitoudegou) [reply migrate-from-parquet-files-to-tidb.md:90]: 这个功能应该没有测试过，不过从代码上来看，应该也可以支持

File: migrate-from-parquet-files-to-tidb.md (hunk 3 above)
Comment (buchuitoudegou) [reply migrate-from-parquet-files-to-tidb.md:80]: 是默认端口

**Comment by ran-huang** [comment]
Thanks!!
I'll revert this commit ([23c2684](https://github.com/pingcap/docs-cn/pull/11261/commits/23c2684ba70ea3095569bbf88fa20f604488c9ed)) after code freeze.

**Review by ran-huang — APPROVED** [review]

File: migration-tools.md (hunk 5 above)
Comment (Oreoxmt) [inline migration-tools.md:18]:
```suggestion
| [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)| 用于将数据全量导入到 TiDB。|Dumpling 输出的文件；CSV 文件；从 Hive 或者 Aurora 导出的 Parquet 文件；从本地盘或 Amazon S3 云盘读取数据。 | TiDB | 支持迅速导入大量新数据，实现快速初始化 TiDB 集群的指定表；支持断点续传；支持数据过滤。| 如果使用 Local-backend 进行数据导入，TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。如果你不希望 TiDB 集群的对外服务受到影响，可以参考 TiDB Lightning TiDB-backend 中的硬件需求与部署方式进行数据导入。|
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**